DepositApi.BLL/DTO/DepositCalcDTO.cs

```diff
@@ -6,10 +6,14 @@ namespace DepositApi.BLL.DTO
 {
     public class DepositCalcDTO
```

Comment:
Using abbreviations such as Calc instead of Calculation or Calculator, not a good idea for naming of the class name.